visualization.rviz

```diff
@@ -124,7 +124,7 @@ Visualization Manager:
     - Class: rviz/Measure
     - Class: rviz/SetInitialPose
       Theta std deviation: 0.2617993950843811
-      Topic: /initialpose
+      Topic: /car0/initialpose
```

Comment:
rename it to something like `single_robot' etc. Having number in this seems confusing.

Reply:
Will do that tmr.

Reply:
done!